chat/command.go

```diff
@@ -190,8 +191,16 @@ func InitCommands(c *Commands) {
 				if cfg.Theme != nil {
 					theme = cfg.Theme.ID()
 				}
-				body := fmt.Sprintf("Current theme: %s", theme)
-				room.Send(message.NewSystemMsg(body, user))
+				var output bytes.Buffer
+				fmt.Fprintf(&output, "Current theme: %s%s", theme, message.Newline)
```

Comment:
Whichever one you prefer I'll go with. I thought using the direct formatting would look better than doing string addition but it's up to you.